setup.cfg

```diff
@@ -5,4 +5,4 @@ test=pytest
 pep8ignore = 
     pythran/tests/*.py ALL
     papers/* ALL
-addopts=--pep8
+addopts=
```

Comment:
mouarf that sould have never landed here... (my py.test installation is not happy with this option!)